### PR TITLE
Add table with examples

### LINKED ISSUE #490: uses/IoT: provide real-life examples similar to uses/online
The "Online Games" page provides examples of successful XMPP deployments with impressive numbers.

We should replace the IoT explanation page with actual examples of XMPP in IoT use.

Known deployments are:

* Google Cloud Print: https://www.google.com/intl/en/cloudprint/learn/ (probably deployed to millions of printers out there, see https://www.google.com/cloudprint/learn/printers/)
* Google Firebase Cloud Messaging: https://firebase.google.com/docs/cloud-messaging/ (deployed to some billion Android devices)
* [Logitech Harmony Hub](https://www.theverge.com/2018/12/21/18152037/logitech-will-restore-third-party-harmony-home-automation)
* ... add your IoT use case here!

### PR DESCRIPTION
**Pull request by Echolon**
I added the table as suggested by @ge0rg 

Closes #490

### CODE REVIEW

**Comment by Echolon** [comment]
@COM8 do you have any other interesting suggestions? Maybe even you thesis to link? :)

**Comment by COM8** [comment]
@Echolon would be possible, since I'm currently working on setting up a `Jura E6 2019 platin (15326)` coffee maker to be fully controllable via my XMPP IoT XEP:
* https://github.com/COM8/esp32-jura

The following repo contains the reference/test implementation for my code.
* https://github.com/COM8/esp32-xmpp-iot

How about something like this:

| Users         | Game / Community                | Description                            | Since |
|-------------|---------------------------------|---------------------------------|-------|
| 1  🤷‍♂️         | [GitHub](https://github.com/COM8/esp32-xmpp-iot) repository | IoT reference implementation for [ESP32](https://www.espressif.com/en/products/socs/esp32/overview) microcontroller ([B.Sc. Thesis](https://home.in.tum.de/~sauterf/html-data/Thesis_Smart_Building_Control_with_XMPP_for_IoT.pdf)). | 2019 |
| 1  🤷‍♂️         | [GitHub](https://github.com/COM8/esp32-jura) repository | `Jura E6 2019 platin (15326)` IoT prototype, controlled by XMPP. | 2020 |

File: content/pages/uses/internet-of-things.md [hunk 1]
```diff
@@ -41,6 +41,17 @@ This definition has some direct consequences that open up into different areas o
 
 As you will see, **XMPP** provides a lot of support into all these areas, making it well suited for use within the area of *Internet of Things*. To check current developments within XMPP and IoT, check the [Work in progress](/uses/iot/work-in-progress) page. There is also a page containing [testimonials](/uses/iot/testimonials).
 
+## Realized example projects
+
+Projects listed which are adopting XMPP for Internet of Things:
+
+| Users         | Game / Community                | Description                            | Since |
```
Comment (ge0rg) [inline content/pages/uses/internet-of-things.md:48]: Maybe "Game / Community" is less appropriate and should be replaced by "System / Application" or something similar.

**Comment by ge0rg** [comment]
It would be good to separate existing deployments (Google GCP, FCM, Logitech) from projects / platforms that can be used by developers for their own projects.

I'd suggest to put the latter into their own table, and in the long term maybe we could create a new category under our implementations page(s).

**Comment by Echolon** [comment]
Yes ok. But I think the list should be first expanded before we separate.

File: content/pages/uses/internet-of-things.md (hunk 1 above)
Comment (Echolon) [reply content/pages/uses/internet-of-things.md]: Fixed!

**Comment by Echolon** [comment]
@Ppjet6 can you review and merge? or should this be reviewed more officially?

**Pull request merged**